pyproject.toml

```diff
@@ -36,6 +36,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.6"
 cyclonedx-python-lib = ">= 2.0.0, < 3.0.0"
+pip-requirements-parser = "^31.2.0"
```

Comment:
What's the lowest version of `pip-requirements-parser` we can work with?

Reply:
As far as I see, there are only [4 versions](https://pypi.org/project/pip-requirements-parser/#history) of pip-requirements-parser published to PyPI and it seems to be based on the pip code itself. The classifiers indicate that it's in [Beta stage](https://github.com/nexB/pip-requirements-parser/blob/f8954c7dfb3eceb99811fd54e243ab91329e11a7/setup.cfg#L18). Based on the [commit history](https://github.com/nexB/pip-requirements-parser/compare/v30.0.0...v31.2.0) and the [changelog](https://github.com/nexB/pip-requirements-parser/blob/main/CHANGELOG.rst), it seems that most of the changes are grooming of the original code and docs of the original [pip](https://github.com/pypa/pip) project. So, I'd say let's stick to the current version by pinning it. WDYT?

Reply:
All good - thanks for checking @mostafa